packages/stylelint-plugin/README.md

```diff
@@ -0,0 +1,77 @@
+# Kaizen Stylelint Plugin
+
+## Tl;dr
+
+`yarn add @kaizen/stylelint-plugin`
+
+Then, add the plugin and the rules you want to your stylelint configuration:
```

Comment:
Hi, I left a comment in #team_foundations (https://cultureamp.slack.com/archives/C016N8Y1E10/p1619501865363200), but I was wondering if you'd consider exporting a base config, so that consumers just need to extend a base config, rather than opt in to each rule individually.

Reply:
replying in slack :)